.github/workflows/test.yml

```diff
@@ -1,30 +1,53 @@
 name: test syntax
 
 on:
+  # Runs on pull requests targeting the default branch
   pull_request:
-    branches:
-    - main
+    branches: [$default-branch]
```

Comment:
```suggestion
    branches: [main]
```

Reply:
Oh, I thought you were being clever 🤣

Reply:
The worse part is that the syntax is valid... for templates...